src/visual.ts

```diff
@@ -0,0 +1,60 @@
+import * as models from 'powerbi-models';
+import { IFilterable } from './ifilterable';
+import { IPage, Page } from './page';
+
+export interface IVisual {
+    name: string;
+    page: IPage;
+}
+
```

Comment:
Instead of the class implementing both interfaces shouldn't the IVisual extend IFilterable?